.github/ISSUE_TEMPLATE/new_blockchain.md

```diff
@@ -24,6 +24,7 @@ Reason:
 - [ ] Coin configuration
```

Comment:
This line seems duplicated, could you please remove it?

Reply:
done